### ISSUE TITLE
IgnoreRewrite

### ISSUE THREAD

**Issue by lncoll** [marcelvanherk/Conquest-DICOM-Server#41]
Found your new addition to dgate, IgnoreRewrite.
I was using a lua script to avoid rewriting files, but this looks better.
But needed to modify slightly, to add a line in logs.
This is as is working fine in my tests:

```
  // newly written image (not rewritten image): use selection preferred storage if enough space  
  if (*rRoot)  
  {  
    if (Storage[0])  
    {  
      int PreferredDevice = atoi(Storage+3);  
      if(CheckFreeStoreOnMAGDevice(PreferredDevice) > 30)  
        sprintf(Device, "MAG%d", PreferredDevice);  
    }  
  }  
  else    
  {  
    rewrite=TRUE;  
    char szRootSC[64], temp[64];  
    if (MyGetPrivateProfileString(RootConfig, "MicroPACS", RootConfig, szRootSC, 64, ConfigFile))  
    {  
      MyGetPrivateProfileString(szRootSC, "IgnoreRewrite", "0", temp, 64, ConfigFile);  
      if (atoi(temp))  
      {  
        OperatorConsole.printf("File already exists: %s\n", rFilename);  
        Filename[0]=0;  
        delete DDOPtr;  
        return TRUE;  
      }  
    }  
  }  

```
Just added the OperatorConsole line and cleaning the filename.
Hope this helps.

Luis

**Comment by marcelvanherk** [comment]
Hi Luis,

why did you need to set Filename[0] ?

Just merging

Marcel

**Comment by marcelvanherk** [comment]
This feature has been released in 1.5.0e
